APMrover2/mode_follow.cpp

```diff
@@ -0,0 +1,82 @@
+#include "mode.h"
+#include "Rover.h"
+
+// initialize follow mode
+bool ModeFollow::_enter()
+{
+    // re-use guided mode
+    gcs().send_text(MAV_SEVERITY_WARNING, "Follow-mode init");
```

Comment:
not needed, specially when you send it even if it failed to enter in follow mode ! ;-)